robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -0,0 +1,147 @@
+
+    Test home phone functionality           ${CONTACT.FirstName}
+    ...                                     ${CONTACT.LastName}
+
+Verify batch functionality of preferred phone
+    Select App Launcher App                 EDA
+    Close all tabs
+
+    Current page should be                  Home           Contacts
+
+    # Verify the EDA Setting 'Disable Preferred Phone enforcement'
+    
+    # The following places us inside an iFrame.  Remember to jump 
+    # back out when done.
+
+    # Make sure the 'Disable Preferred Phone enforcement' checkbox 
+    # has a check in the checkbox
+    Open EDA Settings Tab menu item
+    Enable enchanced checkbox    
+    Set the disable preferred phone enforcement
+    Disable enchanced checkbox    
+    Shift to default content
+
+
+    # Create a new contact and add some phone numbers
+    # Note:  The trigger handler is NOT set to Active, 
+    #        and the 'Disable Preferred Phone enforcement'
+    #        is NOT being enforced
+    Create new contact
+    Select tab                              Contacts
+    # Note:  look in ContactsPageObject.py to see that the following
+    #        test does indeed verify the phone numbers
+    Add home phone to contact and verify    ${CONTACT2.FirstName}
+    ...                                     ${CONTACT2.LastName}
+
+    # Now we'll reconfigure the trigger and EDA Setting
+    # and run another test to check 'Run Cleanup'
+
+    Open EDA Settings Tab menu item
+    Enable enchanced checkbox    
+    Clear the disable preferred phone enforcement
+    Run phone cleanup
+    Capture page screenshot
+    Shift to default content
+
+    # Verify that 'Run Cleanup' produced correct results
+    # The code is found in ContactsPageObject.py
+    Select tab                              Contacts
+    Shift to default content
+    Verify contact values                   ${CONTACT2.FirstName}
+    ...                                     ${CONTACT2.LastName}
+
+    # Restore Settings
+    Open EDA Settings Tab menu item
+    Set the disable preferred phone enforcement
+    Shift to default content
+
+*** Keywords ***
+Initialize test data
+    [Documentation]             Create a contact with a randomly generated firstname and lastname via API
+
+    &{CONTACT} =   API Create Contact
+    #Write To Console            Contact successfully created, with first name: ${CONTACT.FirstName} and last name: ${CONTACT.LastName}
+    Set suite variable          &{CONTACT}
+
+    ${NAMESPACE} =              Get EDA namespace prefix
+    Set suite variable          ${NAMESPACE}
+
+Create new contact
+    [Documentation]             Create a new contact with a randomly generated firstname and lastname via API
+
+    &{CONTACT2} =   API Create Contact
+    #Write To Console            Contact successfully created, with first name: ${CONTACT2.FirstName} and last name: ${CONTACT2.LastName}
+    Set suite variable          &{CONTACT2}
+
+Deactivate trigger handler
```

Comment:
@dashattuck  You commented above in TriggerHandlersPageObject.py that the functionality was removed, so does that mean we don't need `Deactivate trigger handler`, `Enable trigger handler`, and `Get trigger handler` here anymore?

Reply:
Removed all references to the removed code

Reply:
I removed all references to triggers.